GATEWAY/src/pages/applications/applications.ts

```diff
@@ -31,7 +31,17 @@ export class ApplicationsPage {
               private devicesService: DevicesService,
               private mqttClient: MqttClient,
               private tilesApi: TilesApi,
-              private storage: Storage) {}
+              private storage: Storage,
+              private plt: Platform,
+              private alertCtrl: AlertController) {
+
+    if (this.plt.is('android')) {
```

Comment:
Won't this give an alert every time an android user uses the app? There should be a test to see if the location is in fact enabled.